### PR TITLE
Adjusting the size of buttons on the Card

### PR DESCRIPTION
**Pull request by SzymonGorowski**
https://trello.com/c/Cb6hGF1L/662-correction-some-parts-of-ui-to-fit-bigger-screen-sizes

### CODE REVIEW

File: Modules/AirCastingStyling/Sources/AirCastingStyling/FollowButtonStyle.swift [hunk 1]
```diff
@@ -14,7 +14,7 @@ public struct FollowButtonStyle: ButtonStyle {
         configuration
             .label
             .foregroundColor(.white)
-            .frame(width: 82, height: 23, alignment: .center)
+            .frame(width: 82, height: 29, alignment: .center)
             .background(Color.accentColor)
             .font(.muli(size: 13, weight: .semibold))
             .cornerRadius(11)
```
Comment (jankrzempek) [inline Modules/AirCastingStyling/Sources/AirCastingStyling/FollowButtonStyle.swift:20]: Please fix here the cornerRadius as it changed and does not match map and graph button anymore

File: Modules/AirCastingStyling/Sources/AirCastingStyling/UnFollowButtonStyle.swift [hunk 2]
```diff
@@ -14,7 +14,7 @@ public struct UnFollowButtonStyle: ButtonStyle {
         configuration
             .label
             .foregroundColor(.accentColor)
-            .frame(width: 82, height: 23, alignment: .center)
+            .frame(width: 82, height: 29, alignment: .center)
             .background(Color.white)
             .font(.muli(size: 13, weight: .semibold))
             .overlay(
```
Comment (jankrzempek) [inline Modules/AirCastingStyling/Sources/AirCastingStyling/UnFollowButtonStyle.swift:20]: Please fix here the cornerRadius as it changed and does not match map and graph button anymore

**Review by jankrzempek — COMMENTED** [review]
Nice! 😎

File: AirCasting/SessionViews/SessionHeaderView.swift [hunk 3]
```diff
@@ -143,6 +143,9 @@ private extension SessionHeaderView {
                     }) {
                         Image(systemName: chevronIndicator)
                             .renderingMode(.original)
+                            .resizable()
+                            .aspectRatio(contentMode: .fit)
+                            .frame(width: 18, height: 18)
```
[the remark this answers is not in the document]
Comment (SzymonGorowski) [reply AirCasting/SessionViews/SessionHeaderView.swift]: now it's a little bit smaller. Thanks for the advice

File: AirCasting/SessionViews/SessionHeaderView.swift [hunk 4]
```diff
@@ -143,6 +143,9 @@ private extension SessionHeaderView {
                     }) {
                         Image(systemName: chevronIndicator)
                             .renderingMode(.original)
+                            .resizable()
+                            .aspectRatio(contentMode: .fit)
+                            .frame(width: 15, height: 15)
```
Comment (jankrzempek) [inline AirCasting/SessionViews/SessionHeaderView.swift:148]: This one here is mind-blowing, I was trying to accomplish it so hard a long time ago. Big congrats, it's a game changer here 🥳

**Review by jankrzempek — APPROVED** [review]
Pure love for code can be seen here 🤖

**Review by anna1901 — COMMENTED** [review]
I think it would be a good idea to change the height of map and graph buttons for mobile sessions as well. For some reason the graph button on fixed session card is now bigger, but for mobile sessions it stayed small.

**Review by anna1901 — APPROVED** [review]

**Pull request merged**